.github/workflows/promote.yaml

```diff
@@ -0,0 +1,68 @@
+name: Promote release
+
+on:
+  repository_dispatch:
+    types: [promote]
+
+jobs:
+  promote:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12
+          registry-url: 'https://packages.nuxeo.com/repository/npm-public/'
+          scope: '@nuxeo'
+
+      - uses: actions/checkout@v2
+        with:
+          ref: v${{ github.event.client_payload.version }}
+
+      - run: git config user.name "nuxeo-web-ui-jx-bot" && git config user.email "ui+jx-bot@nuxeo.com"
+
+      - run: echo "::set-env name=VERSION::$(echo '${{ github.event.client_payload.version }}' | sed -e 's/-rc.[0-9]*//')"
+
+      - name: Update Nuxeo Elements version to ${{ env.VERSION }}
+        run: |
+          npx lerna version $VERSION --no-push --force-publish --no-git-tag-version --yes
```

Comment:
We also don't have this on the RC script, but shouldn't we also be doing something like this on both:
```
          npm version $VERSION --no-git-tag-version
```
See https://github.com/nuxeo/nuxeo-elements/blob/v3.0.1-rc.2/package.json#L6.

Reply:
we should yes, I'd swear I had added it but apparently only did it when aligning on next snapshot

Reply:
done 👍